atd-vzv/src/views/summary/FatalitiesByTimeOfDay.js

```diff
@@ -129,85 +130,95 @@ const FatalitiesByTimeOfDayWeek = () => {
   return (
     <Container>
       <Row>
-        <h3>Traffic fatalities by time and day</h3>
+        <Col md="2"></Col>
+        <Col md="8">
+          <Nav tabs>
+            <NavItem>
+              <NavLink
+                className={classnames({ active: activeTab === "6" })}
```

Comment:
Be careful with these integers that you are coercing into being strings, seems like a recipe for type confusion. Could they just be integers in state?

Reply:
Switched to integers!